test/test_list_format.py

```diff
@@ -598,5 +598,38 @@ def test_list_format39(self):
 """
         self.assertEqual(self.output, result)
 
+    @mock.patch('topydo.lib.ListFormat.get_terminal_size')
+    def test_list_format40(self, mock_terminal_size):
```

Comment:
`test_list_format40` tests if specifying `\t` in `list_format` option works. Relevant code part is in [`ListFormatParser.__init__()`](https://github.com/bram85/topydo/pull/77/files#diff-c0cff3e2ccdd829757f686fe501da792R97).

`test_list_format41` tests `\t` specified from CLI as an argument for `-F` flag.